src/sparql/kernel/qsparqlbinding.cpp

```diff
@@ -376,13 +376,13 @@ QString QSparqlBinding::toString() const
         {
             quoted = true;
             QDateTime dt = val.toDateTime();
-            int offset = dt.utcOffset();
+            int offset = dt.offsetFromUtc();
```

Comment:
Any idea why QDateTime::toString(Qt::ISODate) isn't used?

Reply:
Maybe https://bugreports.qt.io/browse/QTBUG-26161 but not sure

Reply:
No idea, was here just changing a deprecated method to its replacement which it was already calling internally.